### PR TITLE
Update navigation header to include phase banner

### PR DESCRIPTION
**Pull request by DilwoarH**
screen shot 2018-11-12 at 13 31 20

Ticket: https://trello.com/c/6hvQZTVK/259-update-navigation-header

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -25,12 +25,21 @@
   <%= render "govuk_publishing_components/components/layout_header", {
     environment: GovukPublishingComponents::AppHelpers::Environment.current_acceptance_environment,
     navigation_items: [
+      { text: "Switch app", href: Plek.new.external_url_for("signon") },
```
Comment (DilwoarH) [inline app/views/layouts/application.html.erb]: Need a external url for this

File: app/views/layouts/_phase_banner_elements.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<span class="gem-c-phase-banner__banner-item">
+    <a class="govuk-link govuk-link--no-visited-state" href="https://support.publishing.service.gov.uk/">Send us feedback</a>
```
Comment (DilwoarH) [inline app/views/layouts/_phase_banner_elements.html.erb]: Need to set appropriate URL

File: app/views/layouts/_phase_banner_elements.html.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+</span>
+<span class="gem-c-phase-banner__banner-item">
+    <a class="govuk-link govuk-link--no-visited-state" href="#">What's new</a>
```
Comment (DilwoarH) [inline app/views/layouts/_phase_banner_elements.html.erb]: Need to set appropriate URL

File: app/assets/stylesheets/components/_phase-banner.scss [hunk 4]
```diff
@@ -0,0 +1,10 @@
+.gem-c-phase-banner {
```
Comment (alex-ju) [inline app/assets/stylesheets/components/_phase-banner.scss]: As in the other pull request, this is an overwrite should stay in `app/assets/stylesheets/utilities/_overwrites.scss`

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -25,12 +25,21 @@
...
       { text: "Log out", href: gds_sign_out_path }
     ]
   }%>
 
   <div class="govuk-width-container">
+    <%= render "govuk_publishing_components/components/phase_banner", {
+      app_name: "Content publisher",
+      phase: "beta",
+      message: capture do
+        render "layouts/phase_banner_elements"
```
Comment (alex-ju) [inline app/views/layouts/application.html.erb]: I'm not sure we need a partial for this - we could capture it above - but if we do `layouts` isn't probably the best place. I would check with @benthorner to see where it fits in the project structure.

File: app/views/layouts/_phase_banner_elements.html.erb [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<span class="gem-c-phase-banner__banner-item">
+  <a class="govuk-link govuk-link--no-visited-state" href="<%= Plek.new.external_url_for("support") + "/general_request/new" %>">Send us feedback</a>
```
Comment (alex-ju) [inline app/views/layouts/_phase_banner_elements.html.erb]: We should use translation files for these two pieces of copy.

File: app/views/layouts/application.html.erb (hunk 5 above)
Comment (DilwoarH) [reply app/views/layouts/application.html.erb]: @benthorner - where would you put something like this, I put it in here as it is a generic element used across the whole app.

**Comment by DilwoarH** [comment]
@alex-ju - all changes have been done except for moving the template.

File: app/views/layouts/application.html.erb (hunk 5 above)
Comment (benthorner) [reply app/views/layouts/application.html.erb]: I think I'm more concerned about the footer part, which has made this file long. I think it's fine to make a few partials here, as this file is becoming unreadable for someone trying to get an idea of its structure. And layouts/_partial.html.erb is the place I'd look first to find them - you might need to specify the path explicitly when you render them, though.

**Pull request merged**